docs/operation/upgrade_downgrade.rst

```diff
@@ -0,0 +1,120 @@
+
+.. important::
+
+    - Upgrading and/or downgrading MetalK8s is a manual process and as a result,
+      the following outlined steps must be met to the later in other to achieve
+      a successful upgrade.
+    - The output of the below-provided commands solely depends on the particular
+      MetalK8s setup you have in place.
+    - **<new_version>**, **<current_version>**, **<destination_version>**
+      are suffixes which must be replaced with appropriate version number.
+
+Supported Versions
+******************
+.. note::
+
+    MetalK8 supports upgrade and downgrade **strictly** from one supported
+    minor version to another. For example;
+
+    - Upgrade from 2.0.0 to 2.0.1
+    - Upgrade from 2.0.0 to 2.1.0
+
+    - Downgrade from 2.0.1 to 2.0.0
+    - Downgrade from 2.1.0 to 2.0.0
+
+    Please refer to the release notes for more information.
+
+Upgrade and Downgrade Steps
+***************************
+This section describes the logical steps for Upgrading or Downgrading MetalK8s.
+
+#. Connect to the :term:`Bootstrap node`.
+
+#. Build and copy the new MetalK8 `.ISO` file to the **Bootstrap node**.
+
+#. Edit the :file:`bootstrap.yaml` file located in
+   :file:`/etc/metalk8s/bootstrap.yaml` then add the new MetalK8s ISO location.
+
+   .. code::
+
+      products:
+        metalk8s:
+        - /srv/scality/metalk8s-2.0.0-dev
+        - <path_to_the_new_iso>              <=== New line to add
+
+
+#. Mount the new MetalK8s ISO.
+
+    ::
+
+        salt-call state.sls metalk8s.products.mounted saltenv=metalk8s-<current_version>
+
+            Summary for local
+            ------------
+            Succeeded: 5 (changed=1)
+            Failed:    0
+            ------------
+            Total states run:     5
+
+#. Make the version available to the entire cluster.
+
+   - For upgrade
+
+    ::
+
+        #For upgrade only
+        salt-call --local state.sls metalk8s.products saltenv=metalk8s-<destination_version> pillar="{'metalk8s': {'endpoints': $(salt-call --out txt pillar.get metalk8s:endpoints | cut -c 8-)}}"
+
+   - For downgrade
+
+    ::
+
+        #For downgrade only
+        salt-call --local state.sls metalk8s.products saltenv=metalk8s-<destination_version> pillar="{'metalk8s': {'endpoints': $(salt-call --out txt pillar.get metalk8s:endpoints | cut -c 8-)}}"
+
+#. From the :term:`Bootstrap node`, connect to the :term:`Salt Master`.
+
+    ::
+
+        crictl exec -it $(crictl ps -q --label io.kubernetes.container.name=salt-master) bash
+
+    .. warning::
+        The next few commands must be executed only on the :term:`Salt Master`.
+        Before proceeding, make sure that this is verified.
+
+#. From the :term:`Salt Master`, launch the rolling upgrade/downgrade
+   procedure.
+
+   - For Upgrades
+
+    ::
+
+        #For upgrade only
+        salt-run state.orchestrate metalk8s.orchestrate.upgrade saltenv=metalk8s-<destination_version> pillar="{'orchestrate': {'dest_version': '<new_version>'}}"
```

Comment:
Replace `new_version` by `destination_version` totaly remove the concept of "new version" we only have current and destination.